web/e2e/config/configs.js

```diff
@@ -0,0 +1,124 @@
+// @flow
+
+export type SeleniumCapabilitiesType = {|
+  browserName: string,
+  browserVersion: string,
+  platformName?: string,
+  chromeOptions?: {| args: Array<string> |}
+|}
+
+export type BrowserstackCapabilitiesType = {|
+  os: string,
+  os_version: string,
+  browserName: string,
+  browser_version: string,
+  'browserstack.local': boolean,
+  'browserstack.user': string,
+  'browserstack.key': string,
+  'browserstack.debug': boolean,
+  project: string
+|}
+
+export type EndToEndConfigType = {|
+  url: string,
+  prefix: string,
+  browserstack?: boolean,
+  caps: SeleniumCapabilitiesType | BrowserstackCapabilitiesType
+|}
+
+/*
+  LOCAL
+ */
+
+exports.local_chrome = ({
+  url: 'http://localhost:4444/wd/hub',
+  prefix: 'IG LOCAL',
+  caps: {
+    browserName: 'chrome', // selenium has lowercased browsernames
+    browserVersion: '88'
+  }
+}: EndToEndConfigType)
+
+/*
+  CI
+ */
+
+exports.ci_chrome = ({
+  url: 'http://localhost:4444/wd/hub',
+  prefix: 'IG DEV',
+  caps: {
+    browserName: 'chrome',
+    browserVersion: '88',
```

Comment:
Probably should be removed as we probably do not want to test against different versions of chrome

Reply:
Wouldn't this be helpful, when we get a bug report for a specific browser-version?

Reply:
Removed the browser version for local testing, as probably everyone has a different version. For the ci and browserstack I kept it, as there the version should be fixed (and for browserstack required).